{
	"id": "oval:org.opensuse.security:def:20132132",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"Title": "CVE-2013-2132",
		"affected": {
			"family": "unix",
			"Platform": [
				"openSUSE Tumbleweed"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2013-2132",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-2132",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2013-2132",
				"ref_url": "https://www.suse.com/security/cve/CVE-2013-2132",
				"source": "SUSE CVE"
			},
			{
				"ref_id": "openSUSE-SU-2013:1064-1",
				"ref_url": "https://lists.opensuse.org/opensuse-updates/2013-06/msg00180.html",
				"source": "SUSE-SU"
			}
		],
		"Description": "\n    bson/_cbsonmodule.c in the mongo-python-driver (aka. pymongo) before 2.5.2, as used in MongoDB, allows context-dependent attackers to cause a denial of service (NULL pointer dereference and crash) via vectors related to decoding of an \"invalid DBRef.\"\n    ",
		"advisory": {
			"from": "security@suse.de",
			"Severity": "Moderate",
			"cve": {
				"text": "CVE-2013-2132",
				"href": "https://www.suse.com/security/cve/CVE-2013-2132/",
				"impact": "moderate"
			},
			"bugzilla": [
				{
					"text": "SUSE bug 822798",
					"href": "https://bugzilla.suse.com/822798"
				}
			],
			"issued": {
				"date": "2021-10-02"
			},
			"updated": {
				"date": "2023-02-15"
			},
			"affected_cpe_list": {
				"Cpe": [
					"cpe:/o:opensuse:tumbleweed"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642270",
						"comment": "python-pymongo-3.1.1-1.5 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642271",
						"comment": "python3-pymongo-3.4.0-1.1 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642462",
						"comment": "python36-pymongo-3.11.4-1.2 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642463",
						"comment": "python38-pymongo-3.11.4-1.2 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009642464",
						"comment": "python39-pymongo-3.11.4-1.2 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009634834",
				"comment": "openSUSE Tumbleweed is installed"
			}
		]
	}
}
